timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -893,6 +893,113 @@ def evaluate(
             logger.info(json.dumps(scores_dict, indent=4))
         return scores_dict
 
+    def feature_importance(
+        self,
+        data: Union[TimeSeriesDataFrame, pd.DataFrame, Path, str],
```

Comment:
Make this an optional argument and use validation data (i.e., full `train_data`) saved inside the trainer by default?